lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -56,7 +48,6 @@ def annotate_plugin(span, env, status, response_headers, response_body)
     def trace!(span, zipkin_env, &block)
       trace_request_information(span, zipkin_env)
       span.kind = Trace::Span::Kind::SERVER
-      span.record('whitelisted') if zipkin_env.force_sample?
```

Comment:
This could be potentially breaking change but also potentially misleading, so taking it out, there is an annotation plugin which could bring this again if needed.